docs/docs/connect/data-source/gcs.md

```diff
@@ -36,27 +39,71 @@ If you are using a service account, you will need to run the following command:
 gcloud auth activate-service-account --key-file=path_to_json_key_file
 ```
 :::
-
 You have now configured Google Cloud access from your local environment. Rill will detect and use your credentials the next time you try to ingest a source.
 
-:::tip Did you know?
-If this project has already been deployed to Rill Cloud and credentials have been set for this source, you can use `rill env pull` to [pull these cloud credentials](/connect/credentials/#rill-env-pull) locally (into your local `.env` file). Please note that this may override any credentials you have set locally for this source.
+### Service Account JSON 
+
+Alternatively, we recommend creating explicit credentials in plug into Rill as this will facilitate an easier deploy to Rill Cloud. Assuming you've followed step 1 and 2 above, you'll need to create your json credential with the following command.
```

Comment:
Same comments as on the BigQuery page:
- "in plug into"
- It's unclear what "your json credential" is referring to